src/llvm.cpp

```diff
@@ -1095,6 +1135,10 @@ void emitDeclConstant(Context *ctx, Decl *decl) {
             break;
         }
 
+        case ExprKind_TypeEnum: {
+            break;
```

Comment:
I recommend adding an `emitExprTypeEnum` similar to what is there for Struct's it's mostly there to emit a named type for `-debug` builds